src/redismodule.h

```diff
@@ -210,11 +213,13 @@ typedef struct RedisModuleStreamID {
 #define REDISMODULE_CTX_FLAGS_SERVER_STARTUP (1<<24)
 /* This context can call execute debug commands. */
 #define REDISMODULE_CTX_FLAGS_DEBUG_ENABLED (1<<25)
+/* Trim is in progress due to slot migration. */
+#define REDISMODULE_CTX_TRIM_IN_PROGRESS (1<<26)
```

Comment:
```suggestion
#define REDISMODULE_CTX_FLAGS_TRIM_IN_PROGRESS (1<<26)
```
other flags have `FLAGS_`

Reply:
good catch, thanks